common/src/main/java/edu/snu/nemo/common/ir/edge/executionproperty/DuplicateDataPropertyValue.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2018 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.nemo.common.ir.edge.executionproperty;
+
+import java.io.Serializable;
+
+/**
+ * Value of DuplicateDataProperty.
+ * If decided is false, its physical edge id(edgeId) is not yet discovered.
+ * edgeId will be set in PhysicalPlanGenerator.
```

Comment:
representativeEdgeId is the id of an edge that represents the group with an id of `groupId`.